pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -136,7 +136,7 @@ func calcRestartCountByLogDir(path string) (int, error) {
 	if _, err := os.Stat(path); err != nil {
 		return 0, nil
 	}
-	restartCount := int(0)
+	restartCount := 0
```

Comment:
Would it make sense to move this line down, before the loop where this variable is used?

Reply:
@sourcelliu ^^^^